frontend/src/js/components/devices/dialogs/DeviceConnectionDialog.tsx

```diff
@@ -47,88 +55,97 @@ const docsLinks = [
 ];
 
 const MenderHubReference = () => (
-  <p className="padding-bottom-none">
+  <Typography variant="body1">
     Or visit {/* eslint-disable-next-line react/jsx-no-target-blank */}
     <a href="https://hub.mender.io/c/board-integrations" target="_blank" rel="noopener">
       Mender Hub
     </a>{' '}
     and search integrations for your device and OS.
-  </p>
+  </Typography>
 );
 
 const OnPremDeviceConnectionExplainer = ({ isEnterprise }) => (
   <>
-    <p>
+    <Typography variant="body1">
       You can connect almost any device and Linux OS with Mender, but to make things simple during evaluation we recommend you to get started with a Debian
       based setup. This also works with a Raspberry Pi as a test device.
       <br />
       Follow the <DocsLink path="client-installation/install-with-debian-package" title="installation instructions" /> for Debian packages and select the{' '}
       {isEnterprise ? 'Enterprise' : 'Demo'} server tab to configure the client.
-    </p>
-    <div>For operating system updates, see the documentation to integrate the following with Mender:</div>
-    <ul>
+      <br />
+      For operating system updates, see the documentation to integrate the following with Mender:
+    </Typography>
+    <List>
       {docsLinks.map(item => (
-        <li key={item.key}>
+        <div key={item.key}>
```

Comment:
maybe mui `ListItem` instead of `<div>` ?

Reply:
yeah, I've taken the `div` shortcut to not have to disable the styling adjustments the `ListItem` comes with 😬, should now be a `<ListItem disablePadding className="padding-top-none padding-bottom-none">` - where `disablePadding` doesn't disable all the padding 🤨